instrumentation/build.gradle.kts

```diff
@@ -36,3 +36,8 @@ subprojects {
     instrumentationProjectMuzzle[counter++ % 4].dependsOn(subProj.tasks.named("muzzle"))
   }
 }
+dependencies {
+  testImplementation("org.testng:testng:7.1.0")
+  testImplementation("org.testng:testng:7.1.0")
+  testImplementation("org.testng:testng:7.1.0")
```

Comment:
What's going on here?

Reply:
Good question. I can't recall a reason for adding this. I've removed it.